posthog/tasks/usage_report.py

```diff
@@ -893,6 +895,22 @@ def get_teams_with_rows_synced_in_period(begin: datetime, end: datetime) -> list
     )
 
 
+@timed_log()
+@retry(tries=QUERY_RETRIES, delay=QUERY_RETRY_DELAY, backoff=QUERY_RETRY_BACKOFF)
+def get_teams_with_rows_exported_in_period(begin: datetime, end: datetime) -> list:
+    return list(
+        BatchExportRun.objects.filter(
+            finished_at__gte=begin,
+            finished_at__lte=end,
```

Comment:
to avoid double counting same export in two periods, we should probably do `finished_at__lt=end`?

Reply:
my bad, I got confused with the CH queries where for some of the heavy ones we do split queries (e.g. split the day in 3 parts) and we had some double-counting issues there in the past (there was a fix for it but looking now it still leaves an unresolved edge case - unrelated to this PR though)